### DISCUSSION TITLE
CUE Community Call 2024-03-06

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#2907] [Community Calls]
We are excited to announce our next CUE Community Call for **Wednesday, March 6, 2024 at 1600 UTC.**

Please reply with any questions/topics you would like to see covered.

### Agenda
- Release updates
- Evaluator updates including performance
- WebAssembly & CUE with other languages
- Modules updates
- (alpha.)cuelang.org and Docs and Content
- cuepls / LSP updates
- Community updates

### Meeting details
https://meet.google.com/vkh-rwyd-oxk

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/4y3uC7Pj9C4

### Slides
https://docs.google.com/presentation/d/1AbK1Ou-kBi3jEhT1_QeTSup4_vQhb8B5tFODKKNIbo4/

### Auto-generated transcript
https://docs.google.com/document/d/1OV3XKr1GiVkb87B8Ox5jsjHK4Zwt8OChgc0X_PEJ6AY/

**Comment by fgm** [comment]
Looking for the recording: I'm trying to use the WASM features and am currently stuck with the current instructions from https://docs.google.com/presentation/d/1PaAbydkiG8fp2mapIOYedw_UY1sHz7QG-T7-4tI9e9A/edit#slide=id.p at the last step : Cue doesn't find the WASM file.

**Reply by myitcv** [reply]
@fgm - we'll be uploading the recording today hopefully. For the problem you are seeing with Wasm, please can you [open a new issue](https://github.com/cue-lang/cue/issues/new/choose) with details? @4ad will be happy to help!

**Comment by fgm** [comment]
@myitcv  Sure it's https://github.com/cue-lang/cue/issues/2924

**Comment by dominikdm** [comment]
Added recording, slides and transcript.
FYI: @fgm
